superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -48,6 +48,18 @@ export interface Props {
   actions: Partial<ExploreActions> & Pick<ExploreActions, 'setControlValue'>;
 }
 
+const Button = styled.button`
+  background: none;
+  border: none;
+  text-decoration: underline;
+  color: blue;
```

Comment:
I think Junlin referred to "theme colour blue". I'm not 100% which blue that refers to, but I suspect either the blue of the "Run"/"Save" buttons, or the blue of the "..." and collapse-the-dataset-panel button at the top. Maybe @cccs-RyanS has an idea where to look for theme blues?

Reply:
Oh, my bad. I can find where the exact blue is located.